.github/workflows/coverage.yml

```diff
@@ -21,38 +21,18 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 16
+          cache: "yarn"
```

Comment:
this is a newish flag of setup node that caches yarn deps, we can get rid of a lot of the tasks we had before from below

Reply:
🔥